conda-recipes/dotnetcore-sdk/build.sh

```diff
@@ -0,0 +1,41 @@
+DOTNET_ROOT=$PREFIX/opt/dotnet
```

Comment:
even if we do need to install .net core, wouldn't it be better to have a single script?
this is only for someone building the recipe in their machine, right? the conda package itself, during installation, will not do these things, correct? if so, we can definitely be more relax about what we can expect the developer trying to build the conda recipe has installed